src/pages/LoginGoogle/LoginForm.jsx

```diff
@@ -0,0 +1,73 @@
+import { useState } from "react";
+// import { BtnLoginGoogle } from "./GoogleLogin";
+import { LoginGoogle } from "./LoginGoogle";
+// import { googleLogout, useGoogleLogin } from "@react-oauth/google";
+// import axios from "axios";
+// import PropTypes from "prop-types";
+
+import css from "./LoginForm.module.css";
+// import { styled } from "styled-components";
+
+const LoginForm = () => {
+  const [form, setForm] = useState({
+    email: "",
+    password: "",
+  });
+
+  const handleChange = (event) => {
+    setForm({
+      ...form,
+      [event.target.id]: event.target.value,
+    });
+  };
+
+  const handleSubmit = (event) => {
+    event.preventDefault();
+
+    alert(form.email + " " + form.password);
+  };
```

Comment:
widziałem że logikę od przechodzenia z logowania jest ale puki co samego logowania nie ma, jeśli dałbyś radę to zrobić było by mega
/api/auth/login
/api/auth/register
oba przyjmują tylko email i login
register zwraca tylko potwierdzenie utworzenia konta
login zwraca
 res.status(200).json({
      status: "success",
      data: { email: user.email },
      accessToken,
      refreshToken,
      sid,
    });